spec/lib/msf/core/module_spec.rb

```diff
@@ -46,6 +46,17 @@
     it { is_expected.to respond_to :is_usable }
   end
 
+  describe '#user_data_is_match?' do
+    subject(:msf_module) {
+      msf_module = described_class.new
+      msf_module.user_data = { match: 'match', match_set: 'match_set', run: 'run' }
+      msf_module
+    }
+    specify do
+      expect(msf_module.user_data_is_match?).to eq(true)
```

Comment:
If you rename `user_data_is_match?` to `user_data_match?` then you can use the be_* matchers:

```ruby
specify do
  expect(msf_module).to be_user_data_match
end
```

It is debatable whether that reads properly since that matchness is of the user_data.  I suppose the problem is user_data is a child of the msf_module really and so the match check should be off some helper Module, but that's just me thinking more functional.